index.html

```diff
@@ -0,0 +1,516 @@
+                                <a href=""><img src="assets/image2.jpg" alt="" width="50"></a>
+                                <a href=""><p>'I Married My Best Friend to Avoid Inheritance Tax'</p></a>              
+                            </div>
+                            <div class="col-lg-4 col-md-4 col-sm-12">
+                                <a href=""><img src="assets/image2.jpg" alt="" width="50"></a>
+                                <a href=""><p>'I Married My Best Friend to Avoid Inheritance Tax'</p></a>              
+                            </div>
+                            <div class="col-lg-4 col-md-4 col-sm-12">
+                                <a href=""><img src="assets/image2.jpg" alt="" width="50"></a>
+                                <a href=""><p>'I Married My Best Friend to Avoid Inheritance Tax'</p></a>              
+                            </div>
+                        </div>
+                        <a href=""> <p>CULTURE & TRAVEL</p></a>             
+                        <div class="row">                           
+                            <div class="col-lg-4 col-md-4 col-sm-12">
+                                <a href=""><img src="assets/image2.jpg" alt="" width="50"></a>
+                                <a href=""><p>'I Married My Best Friend to Avoid Inheritance Tax'</p></a>              
+                            </div>     
+                             <div class="col-lg-4 col-md-4 col-sm-12">
+                                <a href=""><img src="assets/image2.jpg" alt="" width="50"></a>
+                                <a href=""><p>'I Married My Best Friend to Avoid Inheritance Tax'</p></a>              
+                            </div>      
+                            <div class="col-lg-4 col-md-4 col-sm-12">
+                                <a href=""><img src="assets/image2.jpg" alt=""  width="50" ></a>
+                                <a href=""><p>'I Married My Best Friend to Avoid Inheritance Tax'</p></a>              
+                            </div>
+                        </div>
+
+                    </div>
+
+
+                    <div>
+                        <p>MORE STORIES</p>
+                        <hr class="mstories-hr">
+                        <div class="d-flex  div-w">
+                            <img class="col-lg-3 mh-100 mw-30 mstories-img" src="assets/image2.jpg" alt="">
+                            
+                            <div class="col-lg-9">
+                                <a href=""><p class="mstories-h">Biden's Biggest Challenge? 67,000 Migrants Stranded at U.S.-Mexico Border</p></a>
+                                <p class="mstories-p">The incoming  not said what he plans to do with the tens of thousands of asylum-seekers living in limbo in crowded shelters and makeshift camps along the border.</p>
+                            </div>
+                        </div>
+                        <hr class="mstories-hr">
```

Comment:
- [ ] The **MORE STORIES** section gets messed up on the tablet and mobile screen sizes

Tablet 👇🏾 
<img width="436" alt="Screen Shot 2021-01-26 at 8 56 27 PM" src=" ps://user-images.githubusercontent.com/48804258/105897819-f6bd1300-6018-11eb-8774-ca1f0d3cd5f5.png">

Mobile 👇🏾 
<img width="409" alt="Screen Shot 2021-01-26 at 8 56 55 PM" src="https://user-images.githubusercontent.com/48804258/105897876-08061f80-6019-11eb-85e8-2afb9196d121.png">